symphony/bdk/core/service/message/message_service.py

```diff
@@ -78,7 +98,7 @@ def __init__(self, messages_api: MultiAttachmentsMessagesApi,
                  attachment_api: AttachmentsApi,
                  default_api: DefaultApi,
                  auth_session: AuthSession):
-        super().__init__(messages_api, auth_session)
+        super().__init__(messages_api, auth_session, message_suppression_api)
         self._message_api = message_api
         self._message_suppression_api = message_suppression_api
```

Comment:
Please remove this line as it is only used in super class

Reply:
Sorry Elias, can you specify what you want removed?  I thought the message_suppression_api needs to be a part of the parent class constructor since the suppress_message endpoint is OBO enabled. Can you clarify? Thanks